src/me/ryanhamshire/PopulationDensity/DataStore.java

```diff
@@ -46,14 +42,7 @@
 import java.io.IOException;
 import java.nio.charset.Charset;
 import java.text.DateFormat;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Collections;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Random;
+import java.util.*;
```

Comment:
no wildcard imports pls

Reply:
Can probably do. I don't believe I ever manually added imports, so I blame IDEA for this. I'll nuke this line in the IDE and see which of the classes or whatever made it think this was necessary. :+1:

Reply:
Still not resolved, fyi.

Reply:
But ya, intellij IDEA does this by default. Can increase default 5 to some large number like 99

Reply:
I finally fixed it, changing that setting to 99 seemed to help. :p